spec/README.md

```diff
@@ -287,6 +287,41 @@ Option 1: Download a pre-built binary
 Option 2: Build from source
 - To build libmongocrypt from source, follow the instructions in the README on the libmongocrypt GitHub repo: https://github.com/mongodb/libmongocrypt
 
+Create AWS KMS keys
+Many of the Field-Level Encryption tests require that you have an encryption
+masterkey hosted on AWS's Key Management Service. Set up a masterkey by following
```

Comment:
Let's use "master key" and `master_key` everywhere we can, i.e. for everything other than calls to the C functions.